core/src/main/java/greencity/webcontroller/ManagementAdvicesController.java

```diff
@@ -0,0 +1,189 @@
+package greencity.webcontroller;
+
+import greencity.constant.HttpStatuses;
+import greencity.dto.PageableDto;
+import greencity.dto.advice.AdvicePostDto;
+import greencity.dto.advice.AdviceVO;
+import greencity.dto.advice.AdviceViewDto;
+import greencity.dto.genericresponse.GenericResponseDto;
+import greencity.service.AdviceService;
+import greencity.service.LanguageService;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import lombok.AllArgsConstructor;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.*;
+import springfox.documentation.annotations.ApiIgnore;
+
+import javax.validation.Valid;
+import java.util.List;
+
+import static greencity.dto.genericresponse.GenericResponseDto.buildGenericResponseDto;
+
+@Controller
```

Comment:
Mark it as RestController and you won't need to add ResponseBody annotation to methods